front/src/components/release-tracklist.tsx

```diff
@@ -28,6 +28,7 @@ import {
 	ListItemText,
 	ListSubheader,
 	Skeleton,
+	Stack,
```

Comment:
I think you can revert all the changes made to this file, as we agreed that icons shouldn't appear here